src/components-styled/sidebar-metric/sidebar-metric.tsx

```diff
@@ -71,10 +71,10 @@ export function SidebarMetric<T extends { difference: unknown }>({
    * we support both but kpi_titel has precedence.
    */
   const title =
-    get(siteText, [localeTextKey, 'kpi_titel']) ||
+    get(siteText, [localeTextKey, 'kpi_titel']) ??
     get(siteText, [localeTextKey, 'titel_kpi']);
 
-  assert(title, `Missing title at ${localeTextKey}.kpi_titel`);
+  assert(title !== undefined, `Missing title at ${localeTextKey}.kpi_titel`);
```

Comment:
Maybe not the time for nitpicking :) 

```suggestion
  assert(isDefined(title), `Missing title at ${localeTextKey}.kpi_titel`);
```